### PR TITLE
feat: Finish UI

### LINKED ISSUE #93: Splash screen looks stretched in older versions of Android
I know the current splash screen isn't the final one, but I wanted to track this issue in case it persists in the final version.

Here's what it looks like on a physical Huawei P8 lite phone (Android 5.0) on Firebase Test Lab, but I can reproduce it on a physical Nexus 4 too:
splash

### LINKED ISSUE #107: Hotspot icon looks red in older versions of Android
Android 5.1 (Nexus 4):

Screenshot_2020-05-29-11-26-34

### LINKED ISSUE #108: Animation in sync with people screen doesn't work well in older versions of Android
Nexus 4: https://drive.google.com/file/d/1wV7MiZIy-2LsFt1HeEIL_HYGOQ7LtOZn/view?usp=sharing

### PR DESCRIPTION
**Pull request by sdsantos**
Closes #93
Closes #108
Closes #107

### CODE REVIEW

**Comment by sdsantos** [comment]
> I tested it locally but it doesn't work. It gets stuck in the loading indicator and nothing happens. Here's what I see in the logs:
> 
> ```
> 2020-05-26 15:42:41.748 16737-16737/tech.relaycorp.courier W/AboutLibraries: Have you missed to call withFields(R.string.class.getFields())? - autoDetect won't work - https://github.com/mikepenz/AboutLibraries/wiki/HOWTO:-Fragment
> 2020-05-26 15:42:41.751 16737-16813/tech.relaycorp.courier W/Adreno-GSL: <gsl_syncobj_dup_fd:268>: 0x7ccdc22be0 dup(74) error 9 Bad file descriptor
> 2020-05-26 15:42:41.762 16737-16737/tech.relaycorp.courier W/ActivityThread: handleWindowVisibility: no activity for token android.os.BinderProxy@9a36374
> 2020-05-26 15:42:41.774 16737-16813/tech.relaycorp.courier W/Adreno-GSL: <gsl_syncobj_dup_fd:268>: 0x7cd2d51b40 dup(74) error 9 Bad file descriptor
> 2020-05-26 15:42:41.792 16737-16813/tech.relaycorp.courier W/Adreno-GSL: <gsl_syncobj_dup_fd:268>: 0x7d634e8ea0 dup(74) error 9 Bad file descriptor
> 2020-05-26 15:42:41.809 16737-16813/tech.relaycorp.courier W/Adreno-GSL: <gsl_syncobj_dup_fd:268>: 0x7ccdc22be0 dup(74) error 9 Bad file descriptor
> ```

Weird, it's like the gradle plugin didn't work and never generated the licenses for you. Can you clear the build and retry?

**Comment by gnarea** [comment]
Weird. Yes, doing a fresh build fixed it for me.

I'm not sure having a short paragraph explaining what Relaynet is is that useful to be honest. I can't think of any scenario where the user wouldn't already know that, so I'd prefer to keep things simple and take it out for now.

On the other hand, I'd suggest to move the about section to the bottom of the screen to make it less prominent. All the apps I've seen (Firefox, Netflix, etc) place it at the bottom too, presumably for the same reason.

Finally, I think "open source licenses" should be replaced with "open source libraries", although that can also be done when we review the final copy.

Wdyt?

**Comment by sdsantos** [comment]
> Weird. Yes, doing a fresh build fixed it for me.
> 
> I'm not sure having a short paragraph explaining what Relaynet is is that useful to be honest. I can't think of any scenario where the user wouldn't already know that, so I'd prefer to keep things simple and take it out for now.
> 
> On the other hand, I'd suggest to move the about section to the bottom of the screen to make it less prominent. All the apps I've seen (Firefox, Netflix, etc) place it at the bottom too, presumably for the same reason.
> 
> Finally, I think "open source licenses" should be replaced with "open source libraries", although that can also be done when we review the final copy.
> 
> Wdyt?

UI's not final, I tried to keep this PR simple just for the licenses, although I did add an About section. Tomorrow I'll review the UI with Cláudia and take in consideration that it could go on the bottom.

I don't think a short description would hurt, or at least linking somewhere to know more. You could have installed the app some months ago to try it out, or got a device with it. A way to learn a bit more about it could help. Maybe even link somewhere to get support, even if it's just an email.

**Comment by gnarea** [comment]
> I don't think a short description would hurt, or at least linking somewhere to know more. You could have installed the app some months ago to try it out, or got a device with it. A way to learn a bit more about it could help. Maybe even link somewhere to get support, even if it's just an email.

I think linking to relaynet.network would be better than displaying a summary of Relaynet or the courier app:

I'm not sure a link or a summary on that screen would help the user who installed the app a few months ago and forgot about it, or someone who got the phone with the app pre-installed. If you find an app that you don't recognise and you open it to see what it's about, I can't imagine most people would care to open the settings before uninstalling it.

However, I do think linking to relaynet.network will be useful for a different reason: I'm drafting [a section for couriers](https://github.com/relaynet/website/blob/role-pages/couriers.md) which I eventually want it to grow into a "Courier Centre", where they can learn more about doing their job safely and efficiently.

**Comment by sdsantos** [comment]
I ended up using the PR for all the UI/UX final touches. It was easier since some of the tasks were inter-dependent.

Main changes:
- New primary color 🟦 
- Several adjustments on margins and text styles across the app
- New sync animation (let us know what do you think)
- New icon and splash screen (temporary, until we have your final version of the icon/logo)

**Comment by sdsantos** [comment]
Closes #93

File: app/build.gradle [hunk 1]
```diff
@@ -145,6 +146,9 @@ dependencies {
     implementation 'org.conscrypt:conscrypt-android:2.4.0'
     implementation 'org.bouncycastle:bcpkix-jdk15on:1.65'
 
+    // Display open-source licenses
+    implementation "com.mikepenz:aboutlibraries:$aboutLibsVersion"
```
Comment (gnarea) [inline app/build.gradle:150]: FYI, the integration may be missing something:

```
2020-05-29 10:32:31.829 5639-5639/tech.relaycorp.courier W/AboutLibraries: Have you missed to call withFields(R.string.class.getFields())? - autoDetect won't work - https://github.com/mikepenz/AboutLibraries/wiki/HOWTO:-Fragment
```

File: app/src/main/res/values/urls.xml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<resources>
+    <string name="relaynet_website">https://relaynet.network/</string>
```
Comment (gnarea) [inline app/src/main/res/values/urls.xml:2]: Can you please make this so localised websites can be used? I'd like to point users on the `es-*` locale to https://relaynet.red/ instead, for example.

Comment (sdsantos) [reply app/src/main/res/values/urls.xml:2]: That's already localizable. If you create a file `values-es/urls.xml` you can write the spanish version there.

Comment (gnarea) [reply app/src/main/res/values/urls.xml:2]: Nice!

**Comment by gnarea** [comment]
BTW, #93 still persists in this PR although it's more subtle now that the image is bigger:
Screenshot_2020-05-29-12-12-16

**Comment by sdsantos** [comment]
Updated icons, fixed splash screen and the libraries warning. I'll address the other issues on a different PR, let's merge this in first.

**Comment by sdsantos** [comment]
Fixed a couple more issues here.

**Comment by gnarea** [comment]
I've just tested the latest changes but #107 and #108 are still present. Were they really meant to be fixed by this PR? I'm happy to fix them in a different PR but in that case we should remove the `closes #107` and `closes #108`.

Also, I think you force-pushed the last commit without pulling first, so #113 is gone. Happy to re-apply it on master.

**Comment by sdsantos** [comment]
> I've just tested the latest changes but #107 and #108 are still present. Were they really meant to be fixed by this PR? I'm happy to fix them in a different PR but in that case we should remove the `closes #107` and `closes #108`.
> 
> Also, I think you force-pushed the last commit without pulling first, so #113 is gone. Happy to re-apply it on master or here before merging -- I don't mind either way.

Forgot to push and forgot you had merged #113 into this branch. I think it's fixed now.

**Pull request merged**